cyences_app_for_splunk/appserver/static/cs_overview.js

```diff
@@ -50,6 +50,8 @@ require([
         { id: 'cs_device_inventory_table', title: 'Device Inventory' },
         { id: 'cs_user_inventory_table', title: 'User Inventory' },
         { id: 'cs_malicious_ip_list', title: 'Malicious IP List' },
+        { id: 'cs_mssql', title: 'MSSQL' },
+        { id: 'cs_oracle', title: 'Oracle' },
```

Comment:
Need to add Database panel on **cs_overview** dashboard to display alerts related to that.